RT/dt_evolve_hpsi.f90

```diff
@@ -61,7 +62,7 @@ subroutine dt_evolve_hpsi
     call update(zfac,ztpsi(:,:,tid),zu(:,ib,ik))
 
 #ifdef ARTED_CURRENT_OPTIMIZED
-    call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
+    if(flag_current) call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
```

Comment:
`if` statement in huge amount of computation loop maybe inhibits the performance.

Is the performance comparable of before code without etrs propagator?